### ISSUE TITLE
A couple test assertions failing on libuv OSX 10.9

### ISSUE THREAD

**Issue by jvburnes** [joyent/libuv#1263]
I cloned origin master with the following log line:

0000000000000000000000000000000000000000 87ae69b6387fcebd75e4d733d3a95f8497d8ca45 <email addr> 1398889391 -0600 clone: from https://github.com/joyent/libuv.git

 and built on OSX 10.9 according to directions.  The build worked fine, but the following assertions failed:

(Not sure what's going on here)
# 

`poll_unidirectional` failed: timeout
# Output from process `poll_unidirectional`: (no output)

[%  77|+ 172|-   1|T   0|S   0]: osx_selectgot some input
with a couple of lines
feel pretty happy

`fs_event_watch_dir` failed: timeout
# Output from process `fs_event_watch_dir`: (no output)

**Comment by saghul** [comment]
Sigh, cannot reproduce those on 10.8 :-/ And nothing changed in poll handles recently :-/ I don't have access to my 10.9 machine now, will check when I get back to it.

**Comment by ibc** [comment]
Soime here on OSX 10.9.3:

```
=============================================================
[%  77|+ 176|-   0|T   0|S   0]: osx_selectgot some input
with a couple of lines
feel pretty happy
[%  86|+ 196|-   0|T   0|S   0]: fs_event_watch_dir
`fs_event_watch_dir` failed: timeout
Output from process `fs_event_watch_dir`: (no output)
=============================================================
[% 100|+ 225|-   1|T   0|S   0]: Done.
FAIL: test/run-tests
=======================================================
1 of 1 test failed
Please report to https://github.com/joyent/libuv/issues
=======================================================
make[2]: *** [check-TESTS] Error 1
make[1]: *** [check-am] Error 2
make: *** [check] Error 2
```

**Comment by saghul** [comment]
Looks like something racey, it failed for me once, then it succeeded twice in a row :-/ Mind taking a look when you have time, @indutny?

**Comment by indutny** [comment]
Yeah, yeah. It is a long time thing. It is just a plain race condition, I think we could just disable it for now, as I don't want to spend much time on it.
